### DISCUSSION TITLE
CAMRAD & FLIGHTLAB

### DISCUSSION THREAD

**Discussion by rhig2000** [wenbinyugroup/.github#11] [VABS Related Discussions] [answered]
Is it possible to pull cross-section properties for both CAMRAD and FLIGHTLAB from the same VABS output file without having to apply transformation computations, post-process? Or should the axis system be defined differently in VABS according on the desired tool convention?

**Comment by wenbinyu-PU** [comment, marked as answer]
We don't know the definitions used by FlightLAB, thus, I don't know the answer. However, beam properties defined any other way can be extracted from VABS output similar as what we did for CAMRAD II.

**Reply by rhig2000** [reply]
Please assume FLIGHTLAB conventions are the same as RCAS.
Is it possible to pull cross-section properties for both CAMRAD and RCAS from the same VABS output file without having to apply transformation computations, post-process? Or should the axis system be defined differently in VABS according on the desired tool convention?

**Reply by wenbinyu-PU** [reply]
If it is the same as RCAS, then it is possible that you can use the 6x6 stiffness matrix and mass matrix from VABS directly. If not, you can already base on the definitions what they provide and extract the properties you needed for a particular code as what we did for CAMRAD II. If you have more specific questions, please ask.

**Reply by rhig2000** [reply]
Ok, thanks. I'm asking because I suspect that for CAMRAD, the VABS user-defined coordinate system need to be aligned to the blade frame in order that THETAI and THETAC angles include TWISTA, whereas for FLIGHTLAB/RCAS, the VABS user-defined coordinate system need to be aligned orthogonal to the chord. Hence, (2) separate VABS output files are required. The difference between the (2) coordinate systems being TWISTA, the angle from the blade frame to the airfoil chord, which is not related to either inertia or stiffness, but rather to aerodynamic properties. Then when it comes to recovery analysis, CAMRAD and FLIGHTLAB/RCAS loads will again be in different axes systems but will correspond to their respective VABS models. Unfortunately, it seems we'll need to manage (2) VABS models for both tools. They're not compatible with each other. Am I getting this right?
